app/views/patients/show.html.erb

```diff
@@ -47,6 +47,9 @@
           <div class="level-item">
```

Comment:
There should be another header on patient show page for Lab Results, just like there is for immunizations. Right now there is nothing specifying that those resources are lab results, and if you add a lab result and an immunization, the lab results are placed under the immunizations header.

Reply:
resolved